mypyc/ops.py

```diff
@@ -544,6 +567,41 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_load_int(self)
 
 
+class ModuleGet(RegisterOp):
```

Comment:
`ModuleGet` is a little confusing as a name. Maybe rename to `LoadModule`, for consistency with `LoadNone` etc.?